### PR TITLE
add html homepage

### LINKED ISSUE #3: 03_create_html_homepage

### PR DESCRIPTION
**Pull request by xsenikx**
close #3

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    <link rel="stylesheet" href="./assets/css/home.css" />
+    <script
+      src="https://kit.fontawesome.com/351c0f0315.js"
+      crossorigin="anonymous"
+    ></script>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&family=Shrikhand&display=swap"
+      rel="stylesheet"
+    />
+    <title>Oh My Food</title>
```
Comment (Tutt-J) [inline index.html:24]: Mising meta description

File: index.html [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  </head>
+  <body>
+    <div>
```
Comment (Tutt-J) [inline index.html]: What is this div ?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+    </div>
+    <header>
+      <a href="index.html">
+        <img src="./assets/img/logo/ohmyfood@2x.svg" alt="logo" />
+      </a>
+    </header>
+    <main>
+      <section id="location">
+        <i class="fas fa-map-marker-alt"></i>
+        <h1>Paris, Belleville</h1>
```
Comment (Tutt-J) [inline index.html]: Not a h1

File: index.html [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+      </section>
+      <section id="search">
+        <h2>
```
Comment (Tutt-J) [inline index.html]: H1

File: index.html [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+          Réservez le menu qui <br />
```
Comment (Tutt-J) [inline index.html]: Remove br

File: index.html [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+          vous convient
+        </h2>
+        <p>
+          Découvrir des restaurants d'exception,<br />
```
Comment (Tutt-J) [inline index.html]: Remove br

Comment (Tutt-J) [inline index.html]: Découvrez

File: index.html [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+          sélectionnés par nos soins.
+        </p>
+        <button>Explorer nos restaurants</button>
```
Comment (Tutt-J) [inline index.html]: Not a button it's a `<a>`

File: index.html [hunk 8]
```diff
@@ -0,0 +1,168 @@
...
+      </section>
+      <section id="fonctionnement">
+        <h2>Fonctionnement</h2>
+        <div>
```
Comment (Tutt-J) [inline index.html]: Use ol with li, not put number we put it later with css before

File: index.html [hunk 9]
```diff
@@ -0,0 +1,168 @@
...
+          </div>
+          <div>
+            <p>3</p>
+            <i class="fas fa-store"></i>
+            <p>Dégustez au restaurant</p>
+          </div>
+        </div>
+      </section>
+      <section id="restaurant">
+        <h2>restaurants</h2>
```
Comment (Tutt-J) [inline index.html]: Missing maj

File: index.html [hunk 10]
```diff
@@ -0,0 +1,168 @@
...
+        <div>
```
Comment (Tutt-J) [inline index.html]: Remove div

File: index.html [hunk 11]
```diff
@@ -0,0 +1,168 @@
...
+                  src="./assets/img/restaurants/La-palette-du-goût.jpg"
+                  alt="La-palette-du-goût"
+                />
+              </figure>
+              <div>
+                <h3>La palette du goût</h3>
+                <p>Ménilmontant</p>
+              </div>
+              <div>
+                <i class="fas fa-heart"></i>
```
Comment (Tutt-J) [inline index.html]: Not the same heart two times

File: index.html [hunk 12]
```diff
@@ -0,0 +1,168 @@
...
+                <i class="far fa-heart"></i>
+              </div>
+              <div>
+                <p>Nouveau</p>
```
Comment (Tutt-J) [inline index.html]: Mus be on figure with img

**Review by Tutt-J — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (xsenikx) [reply index.html]: it's for my animation "loading-page"

**Review by Tutt-J — APPROVED** [review]

**Pull request merged**